dts/bindings/display/nxp,imx-elcdif.yaml

```diff
@@ -14,5 +14,87 @@ properties:
     interrupts:
       required: true
 
-    label:
+    hsync:
+      type: int
       required: true
+      description: HSYNC pulse width
+    hfp:
+      type: int
+      required: true
+      description: Horizontal front porch
+
+    hbp:
+      type: int
+      required: true
+      description: Horizontal back porch
+
+    vsync:
+      type: int
+      required: true
+      description: VSYNC pulse width
+
+    vfp:
+      type: int
+      required: true
+      description: Vertical front porch
+
+    vbp:
+      type: int
+      required: true
+      description: Vertical back porch
```

Comment:
For all of these: units?

Reply:
Thank you, I have added the units in the description